cmd/zfs/zfs_main.c

```diff
@@ -6859,7 +6859,7 @@ print_holds(boolean_t scripted, int nwidth, int tagwidth, nvlist_t *nvl,
 			if (scripted) {
 				if (parsable) {
 					(void) printf("%s\t%s\t%ld\n", zname,
-					    tagname, time);
+					    tagname, (long)time);
```

Comment:
As I said in another PR, `time_t` is `__int64_t` on 32-bit platforms and so I think this should be:
```suggestion
				(void) printf("%s\t%s\t%lld\n", zname,
					    tagname, (long long)time);
```